Description: Upstream changes introduced in version 2.2.0.2-1ubuntu2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 dkms (2.2.0.2-1ubuntu2) oneiric; urgency=low
 .
   * Prevent crash in apport hook if the package is not an Ubuntu one
     (LP: #816199)
 .
 The person named in the Author field signed this changelog entry.
Author: Brian Murray <brian@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/816199

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- dkms-2.2.0.2.orig/dkms_apport.py
+++ dkms-2.2.0.2/dkms_apport.py
@@ -48,9 +48,12 @@ if options.kernel:
     # TODO: Ubuntu specific
     kernel_package = "linux-headers-" + options.kernel
 
-    if not apport.packaging.is_distro_package(kernel_package):
-        print >> sys.stderr, 'ERROR (dkms apport): kernel package %s is not supported' % (kernel_package,)
-        sys.exit(1)
+    try:
+        apport.packaging.is_distro_package(kernel_package)
+    except ValueError, e:
+        if e == 'package does not exist':
+            print >> sys.stderr, 'ERROR (dkms apport): kernel package %s is not supported' % (kernel_package)
+            sys.exit(1)
 
 make_log=os.path.join('/var','lib','dkms',options.module,options.version,'build','make.log')
 
